.travis.yml

```diff
@@ -1,20 +1,30 @@
 ---
 language: csharp
 
-mono:
- - 4.6.1
- - latest # => "stable release"
- - weekly # => "latest commits"
+matrix:
```

Comment:
We primarily build on mono (while .Net Core has been maturing), so it's important to us to see edge cases on the mono runtime in CI before we hit them during normal development. I've established a routine to notify in the mono chat room every time they break something in latest/weekly, which has seemingly worked and they are now saying that the current mono is the most stable ever. I do the same thing in Logary and Suave.